src/Standards/Squiz/Tests/Commenting/PostStatementCommentUnitTest.inc

```diff
@@ -52,3 +52,7 @@ $match = match($foo // comment
 ) {
     1 => 1, // comment
 };
+
+$a = 1; // @codeCoverageIgnore
+$b = 2; //@phpstan-ignore variable.undefined
+$c = 3; //  @thisIsNotAnAnnotation - more than one space before the @ symbol.
```

Comment:
Just realized, we're also still missing a test with a comment starting with an `@`, but without tag, so `// @ = add tag` (or something like that), which should not flagged as an annotation, but as a normal comment.

Reply:
I just added those two tests.